contrib/windows_scripts/cubrid_env.bat

```diff
@@ -34,8 +34,8 @@ reg add "HKEY_LOCAL_MACHINE\SOFTWARE\CUBRID\CUBRID" /v "Patch" /t REG_SZ /d "0"
 
 reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" /v "CUBRID" /t REG_SZ /d "%CUBRID%\\" /f
 reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" /v "CUBRID_DATABASES" /t REG_SZ /d "%CUBRID_DATABASES%" /f
-reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" /v "Path" /t REG_SZ /d "%CUBRID%\bin;%PATH%" /f
+reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" /v "Path" /t REG_SZ /d "%CUBRID%\cci\bin;%CUBRID%\bin;%PATH%" /f
 
 echo %CUBRID%
 echo %CUBRID_DATABASES%
-echo %Path%
\ No newline at end of file
+echo %Path%
```

Comment:
Add a newline at the end of the file.

Reply:
changed file has eol(end of the line) but original file is noeol (not have eol).

Reply:
It was my mistake.